crates/uv/src/lib.rs

```diff
@@ -58,17 +58,17 @@ pub(crate) mod version;
 #[instrument(skip_all)]
 async fn run(cli: Cli) -> Result<ExitStatus> {
     // Enable flag to pick up warnings generated by workspace loading.
-    if !cli.global_args.quiet {
+    if !cli.top_level.global_args.quiet {
         uv_warnings::enable();
     }
 
     // Switch directories as early as possible.
-    if let Some(directory) = cli.global_args.directory.as_ref() {
+    if let Some(directory) = cli.top_level.global_args.directory.as_ref() {
         std::env::set_current_dir(directory)?;
     }
 
     // The `--isolated` argument is deprecated on preview APIs, and warns on non-preview APIs.
-    let deprecated_isolated = if cli.global_args.isolated {
+    let deprecated_isolated = if false {
```

Comment:
This point needs a better solution for the deprecated `--isolated` - need your input.